guide/comments.md

```diff
@@ -1,29 +1,46 @@
 # Comments
 
+Comments in Fluent start with `#`, `##`, or `###`, and can be used to
+document messages and to define the outline of the file.
+
+Single-hash comments (`#`) can be standalone or can be bound to messages. If
+a comment is located right above a message it is considered part of the
+message and localization tools will present the message and the comment
+together. Otherwise the comment is standalone (which is useful for commenting
+parts of the file out).
+
+Double-hash comments (`##`) are always standalone. They can be used to divide
+files into smaller groups of messages related to each other; they are
+group-level comments. Think of them as of headers with a description.
+Group-level comments are intended as a hint for localizers and tools about
+the layout of the localization resource.
```

Comment:
Can you add docs on how to end a group? Aka, something like

    The grouping ends with the next group comment. Use an empty group comment if there's no following group.

Or something not that butchered.